library/manageiq_custom_attributes.py

```diff
@@ -150,6 +150,15 @@ def update_custom_attribute(self, entity_type, entity_id, ca, ca_href):
         except Exception as e:
             self.module.fail_json(msg="Failed to update the custom attribute {ca_name}. Error: {error}".format(ca_name=ca['name'], error=e))
 
+    @staticmethod
```

Comment:
I like the change, but it means that we are assuming that a custom attribute's section cannot be updated (it will add a new one with a different section but same name and value).
If I'm right, are we comfortable with that?
@cben @simon3z